cosmoz-bottom-bar.js

```diff
@@ -107,7 +121,7 @@
 
 			_computedBarHeight: {
 				type: Number,
-				computed: '_computeComputedBarHeight(matchElementHeight, barHeight, _computedBarHeightKicker)',
```

Comment:
I assume you asked whether to add this kicker back again, and unless you can tell me why it's not needed or how it's solved some other way, yes.